src/utils.ts

```diff
@@ -233,4 +232,30 @@ export function arraysEqual(a: any[], b: any[]) {
       if (a[i] !== b[i]) return false;
     }
     return true;
-  }
\ No newline at end of file
+  }
+
+const alphabet = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789".split("");
+const base = alphabet.length;
+export function encode(id: number) {
```

Comment:
Thoughts on using an encryption algorithm to generate 6 or 8 character hashes? Would add a sense of privacy and uniformity when saving strats at the cost of a very small chance of collision.

Reply:
Rather than do something more involved, I've set the linkID to be randomly chosen from 0 to 62^8 before checking for collisions. Since collisions will be pretty rare, we probably won't run into any problems with this, and link hashes shouldn't be predictable short of someone abusing the fact that we're not using a secure RNG.